test/strategies/sonatype/index.unit.js

```diff
@@ -132,3 +133,30 @@ test("sonatype strategy: hydratePayloadDependencies when using NodeSecure standa
   restoreHttpAgent();
   tape.end();
 });
+
+
+test("sonatype strategy: fetchDataForPackageURLs with chunked coordinates", async(tape) => {
+  const [mockedHttpAgent, restoreHttpAgent] = setupHttpAgentMock();
+  const mockedHttpClient = mockedHttpAgent.get(kSonatypeOrigin);
+
+  let chunkedCoordinates = []
```

Comment:
I would not call that `chunkedCoordinates` here as they are not chunked yet, also you could do that to generate your collection of coordinates:

```js
const coordinates = Array.from({ length: 200 }, () => kFakePackageURL);
```

Reply:
done =)